docs/release-notes/8.10.asciidoc

```diff
@@ -1,10 +1,66 @@
 [[release-notes-header-8.10.1]]
 == 8.10
 
+[discrete]
+[[release-notes-8.10.2]]
+=== 8.10.2
+
+[discrete]
+[[known-issue-8.10.2]]
+==== Known issues
+
+* The alert prevalence feature behaves inconsistently and may not produce accurate results.
```

Comment:
Let's add a quick description of what alert prevalence is and a docs `xref` link for context.

Reply:
Sure, not a problem. How about the following:

```suggestion
* The alert prevalence feature shows whether data from the alert was frequently observed on other hosts and user events. In this release, the feature behaves inconsistently and may not produce accurate results.
```